minvws/nl-kat-coordination
# Added `reports.md` to the documentation tree

### Changes
By adding `reports.md` to the toctree and changing the first line to not have spaces it gets included inside the documentation.

### Issue link
Closes #3111

### Demo

#### Before
![image](https://github.com/minvws/nl-kat-coordination/assets/34127049/968a30c3-afd2-4d09-86cb-ce88e191b1e8)

#### After
![image](https://github.com/minvws/nl-kat-coordination/assets/34127049/b34fff12-c1e6-4b0b-bf21-19919aec48bc)


### QA notes
```mermaid
flowchart TD
    A{START} --> |Go to\nhttps://docs.openkat.nl| B
    B(OpenKAT\ndocumentation) -->|Go to \n`Developer documentation`| C
    C(Developer documentation) -->|Go to `Reports`| D
    D{{fa:fa-check Reports}}
```
If you are able to go to reports, then the problem has been fixed successfully.

---

### Code Checklist







---

## Checklist for code reviewers:


---

## Checklist for QA:


### What works:

- _bullet point + screenshot (if useful) per tested functionality_

### What doesn't work:

- _bullet point + screenshot (if useful) per tested functionality_

### Bug or feature?:

- _bullet point + screenshot (if useful) if it is unclear whether something is a bug or an intended feature._
